version/news/NEWS-2024.08.0-cranberry-hibiscus.md

```diff
@@ -95,6 +96,13 @@
 - Fixed bug that prevented Workbench from respecting licenses with an unlimited named user limit when re-activating SCIM provisioned users (rstudio-pro#6561)
 - Fixed bug where user provisioning would not work properly if the homedir column in the database for any user was null (rstudio-pro#6587)
 - Fixed a pagination issue in the SCIM API that could prevent more than 100 users from being provisioned via Okta (rstudio-pro#6608)
+- Fixed various bugs that could cause NSS requests to fail when automatic user provisioning is enabled (rstudio-pro#6676, rstudio-pro#6657)
+
+### Upgrade Instructions
+
+#### Posit Workbench
+
+- If upgrading from an installation of Workbench 2024.04 with automatic user provisioning enabled, the `server-address` setting in `workbench-nss.conf` should be updated to `unix:/var/run/rstudio-server/rstudio-rserver/user-service.socket`. See [User Provisioning Configuration in the Workbench Admin Guide] for more information. (rstudio-pro#6591)
```

Comment:
What is the consequence if this is not changed? Will it fail to function, or just affect stability?  It would be helpful to include a statement along the lines of "Making this change will... " or "Failure to make this change will..."

Reply:
Addressed with [Jeff's suggestion](https://github.com/rstudio/rstudio/pull/15078#discussion_r1730395758)